.github/workflows/long-running-azure.yaml

```diff
@@ -82,9 +83,9 @@ env:
   VALID_RADIUS_BUILD_WINDOW: 86400
 
   # The AKS cluster name
-  AKS_CLUSTER_NAME: "radiuse2e00-aks"
+  AKS_CLUSTER_NAME: "radiuse2e000-aks"
```

Comment:
Where do we get this name from?

Reply:
This is new cluster name. We can define whatever we want. I deployed this cluster using the bicep template in /test/infra/azure. I will document it in on-call wiki page once this change is completed.

Reply:
I see. Could you also include this information in the comment [here](https://github.com/radius-project/radius/blob/24de5f5d0f9537c66685a78f825ac60bc0760e0d/.github/workflows/long-running-azure.yaml#L85) to make it easier for anyone looking at it to understand.

Reply:
Also, could we give it a more explicit name, something like longhaultest? What is the significance of `2e000`?

Reply:
I will create new one, `rad-longrunningtest-00`. `00` is the sequence number of cluster for the case when we want to migrate.

Reply:
There is the resource name limitation for grafana dashboard and loganalytics. We can use only alphabet + numbers but we cannot exceed 13 lengths. So I created new cluster with `radlrtest00`.

Reply:
sounds good, thanks!